### DISCUSSION TITLE
Enable GitHub Copilot in Android Studio

### DISCUSSION THREAD

**Discussion by GabeGiro** [community/community#42761] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I continuously get a variation of: GitHub Copilot: The login to GitHub failed. Reason: Error: EACCES: permission denied, open '/Users/gabrielgircenko/.config/github-copilot/hosts.json', request id: 6, error code: 1001 
after I try to login to GitHub Copilot through Android Studio. Please, help me setup GitHub Copilot in Android Studio.

**Comment by Akash1134** [comment]
Hi @GabrielGircenko, Is it possible that the internet service provider may be intercepting your SSL traffic or you are using a proxy server to connect?

**Reply by GabeGiro** [reply]
@Akash1134 I had NordVPN on and off, also I was physically in 2 different countries so far, so I don't think this is an issue

**Reply by GabeGiro** [reply]
@Akash1134 my chat with openai on this matter might be helpful too https://chat.openai.com/chat/663174cd-68fc-4458-88bb-71a1593c1c8e
